### PR TITLE
Modify GetFingerprintingIntervalIndices to improve reliability

### PR DESCRIPTION
**Pull request by tlangs**
We've run into some issues with `xargs` in WGS joint genotyping in this step. The simplified bash using `for` loops seems to fix these issues.

### CODE REVIEW

**Review by snovod — APPROVED** [review]
Thank you!

**Comment by kbergin** [comment]
I'm sorry we didn't see this PR sooner - def feel free to ping us if we ever miss a PR from you!

Is this change included in @snovod 's PR #224 or does it still need to be merged in separately?

File: tasks/broad/JointGenotypingTasks.wdl [hunk 1]
```diff
@@ -999,13 +999,19 @@ task GetFingerprintingIntervalIndices {
...
 
-    cat ~{write_lines(unpadded_intervals)} | xargs -n1 basename | xargs -I{} bash -c 'rename_scatter $@' _ {}
+    while read line; do
+      name=$(basename $line)
+      rename_scatter $name
+    done < ~{write_lines(unpadded_intervals)}
 
     #find the first header
-    find . -name "scattered.renamed.*.interval_list" | head -n1 | xargs cat | grep '^@' > all.interval_list
+    first_header_file=$(find . -name 'scattered.renamed.*.interval_list' -print -quit)
```
Comment (ldgauthier) [inline tasks/broad/JointGenotypingTasks.wdl:1008]: We could benefit from some local variables here.  I'm a little anxious about hard coding "scattered.renamed" in two places.  If you're going to make that change you might want to add "@" as "interval_list_comment_char" or something.

**Review by ldgauthier — APPROVED** [review]
Sorry this fell through the cracks. I'm glad to get those xargs commands out.  I didn't go over the by_chromosome changes, but one small comment on fingerprinting task.

**Comment by gbggrant** [comment]
@ldgauthier are these changes still needed?

**Comment by gbggrant** [comment]
Hi @ldgauthier @kbergin is this PR necessary anymore?

**Pull request closed without merging**